build_lookup.sh

```diff
@@ -19,5 +19,9 @@ rm -rf ./txblocks.db
 rm -rf ./test.db
 rm -rf ./txbodies.db
 
+find . -name *.cmake | xargs rm -rf
+find . -name CMakeLists.txt | xargs rm -rf 
```

Comment:
❗️ This line breaks the build by deleting the CMake input.

Reply:
I corrected it. It should have been CMakeCache. Thanks.